AnalyzerTools/src/MCCorrection.C

```diff
@@ -425,6 +425,8 @@ double MCCorrection::MuonTrigger_Eff(TString ID, TString trig, int DataOrMC, dou
 
 double MCCorrection::MuonTrigger_SF(TString ID, TString trig, const std::vector<Muon>& muons, int sys){
 
+  if(muons.size() == 0) return 1.;
+
```

Comment:
@sansan9401 this is needed otherwise code can enter '  if(trig=="IsoMu24" || trig=="IsoMu27" || trig=="Mu50"){' and return 'nan'